test/prefer-math-trunc.js

```diff
@@ -35,6 +39,16 @@ ruleTester.run('prefer-math-trunc', rule, {
 			errors: [error],
 			output: 'const foo = Math.trunc(111);'
 		},
+		{
+			code: 'const foo = (1 + 2 / 3.4) | 0;',
+			errors: [error],
+			output: 'const foo = Math.trunc(1 + 2 / 3.4);'
+		},
+		{
+			code: 'const foo = bar((1.4 | 0) + 2);',
+			errors: [error],
+			output: 'const foo = bar((Math.trunc(1.4)) + 2);'
```

Comment:
Do you know how I can make it so it does not require the parenthesis around `Math.trunc(1.4)` @fisker ?

Reply:
I think we can keep it.